docs/en/operation/User-CLI.md

```diff
@@ -882,6 +887,8 @@ $ ./bin/alluxio fs distributedLoad /data/today --locality-file /tmp/localityfile
 $ ./bin/alluxio fs distributedLoad /data/today --excluded-locality ROCK1,ROCK2
 # Include all workers except which's locality belong to the localities in the excluded locality file
 $ ./bin/alluxio fs distributedLoad /data/today --excluded-locality-file /tmp/localityfile-exclude
+# Turn on async submission mode
```

Comment:
nit: extra space at the start of the line that was not present in most of the above lines